### PR TITLE
Thin Pipfile and Lock

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,9 +5,9 @@ name = "pypi"
 
 [packages]
 # Wheels for cryptography until the next release.
-cryptography35_linux = { file='./wheelhouse/cryptography-2.3.dev1-cp35-cp35m-linux_x86_64.whl', markers="python_version >= '3.5' and python_version < '3.6' and platform_system == 'Linux'"}
-cryptography36_linux = { file='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-linux_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Linux'"}
-cryptography36_darwin = { file='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-macosx_10_13_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Darwin'"}
+cryptography35_linux = { path="./wheelhouse/cryptography-2.3.dev1-cp35-cp35m-linux_x86_64.whl", markers="python_version >= '3.5' and python_version < '3.6' and platform_system == 'Linux'"}
+cryptography36_linux = { path="./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-linux_x86_64.whl", markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Linux'"}
```
Comment (michwill) [inline Pipfile]: Hah, btw.. Does path work with urls?

Comment (KPrasch) [reply Pipfile]: Here's the best I can find :-) https://github.com/pypa/pipenv/issues/2418

Comment (michwill) [reply Pipfile]: Ah, I see. And also, file didn't work for remote uris..

Comment (KPrasch) [reply Pipfile]: Yes. Are you thinking we switch to a remote source?

**Pull request merged**